apps/docs/content/3-components/2-library/progress-bar.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: Progress Bar
+description: Progress Bar HTML Component
+status: in-review
+libraries:
+  - platform: 'local'
+    status: 'not-available'
+  - platform: 'global'
+    link: '?path=/docs/components-progressbar--docs'
+    status: 'beta'
+  - platform: 'react'
+    link: '?path=/docs/components-progressbar--docs'
+    status: 'beta'
+---
+
+# Progress Bar
+
+Use the progress bar component to visually represent the progress of a task or operation.
+
+## Variants
+
+### Progress Bar
+
+<ComponentContainer>
+  <ProgressBar currentValue="40" finalValue="100" size="sm" />
+</ComponentContainer>
+
+### Indeterminate Progress Bar
+
+<ComponentContainer>
+  <ProgressBar
+    isIndeterminate="true"
+    label="Loading..."
+    size="sm"
+    color="blue"
+  />
+</ComponentContainer>
+
+<Tabs id="tab1">
+  <TabList>
+    <TabItem value="tab11" checked>HTML</TabItem>
+    <TabItem value="tab12">Macro</TabItem>
+    <TabItem value="tab13">React</TabItem>
+  </TabList>
+  <TabPanel value="tab11">        
+    ```html
+      <div class="gi-progress-bar-container">
+        <div role="progressbar" aria-valuenow="40" aria-valuemin="0" aria-valuemax="100" aria-labelledby="progress-bar-:S1:" aria-label="Progress bar" class="gi-progress-bar gi-h-2 gi-bg-emerald-800">
+          <div class="gi-h-2 gi-bg-blue-500" style="width: 40%;">
+          </div>
+        </div>
+      </div>
+    ```
+  </TabPanel>
+  <TabPanel value="tab12">        
+    ```html
+      {{ govieProgressBar({"label": "Loading...", "size": "md", "color": "green", "currentValue": 50 }) }}
+    ```
+  </TabPanel>
+  <TabPanel value="tab13">
+    ```html
+      import { ProgressBar } from '@govie-ds/react';
+
+      <ProgressBar label="Loading..." size="sm" />
+
+      <ProgressBar isIndeterminate />
+    ```
+
+  </TabPanel>
+</Tabs>
+
+## Usage
```

Comment:
I am currently following a practice where I am adding Tabs below Usage heading. Is that the expectations or Usage heading requires content?

@pregno

Reply:
@hamza14khan do you have an example? 
All components I took a look it is above including Pagination.
Not sure if I'm missing something.

Reply:
Use the tabs with the snippet code below each variant.
The `Usage` paragraph is intended for group best practices, do's and don'ts